tests/test-cmd/test.build.js

```diff
@@ -227,6 +227,15 @@ describe('build', () => {
       assert.equal(defaultFilter.wantFile('path/to/node_modules'), false);
     });
 
+    it('ignores the artifacts dir', () => withTempDir(
+      (tmpDir) => {
+        const filter = new FileFilter({
+          filePathsToIgnore: [path.join(tmpDir.path(), 'artifacts')],
```

Comment:
I'd like to see one other array item in this list for the purposes of the test. Then you can add an additional `filter.wantFile(...)` assertion to make sure both array items return false.